packet_forwarder/global_conf.json.sx1250.EU868_B

```diff
@@ -0,0 +1,121 @@
+        },
+        "radio_0": {
+            "enable": true,
+            "type": "SX1250",
+            "freq": 867500000,
+            "rssi_offset": -215.4,
+            "rssi_tcomp": {"coeff_a": 0, "coeff_b": 0, "coeff_c": 20.41, "coeff_d": 2162.56, "coeff_e": 0},
+            "tx_enable": true,
+            "tx_freq_min": 863000000,
+            "tx_freq_max": 870000000,
+            "tx_gain_lut":[
+                {"rf_power": 12, "pa_gain": 1, "pwr_idx": 4},
+                {"rf_power": 13, "pa_gain": 1, "pwr_idx": 5},
+                {"rf_power": 14, "pa_gain": 1, "pwr_idx": 6},
+                {"rf_power": 15, "pa_gain": 1, "pwr_idx": 7},
+                {"rf_power": 16, "pa_gain": 1, "pwr_idx": 8},
+                {"rf_power": 17, "pa_gain": 1, "pwr_idx": 9},
+                {"rf_power": 18, "pa_gain": 1, "pwr_idx": 10},
+                {"rf_power": 19, "pa_gain": 1, "pwr_idx": 11},
+                {"rf_power": 20, "pa_gain": 1, "pwr_idx": 12},
+                {"rf_power": 21, "pa_gain": 1, "pwr_idx": 13},
+                {"rf_power": 22, "pa_gain": 1, "pwr_idx": 14},
+                {"rf_power": 23, "pa_gain": 1, "pwr_idx": 15},
+                {"rf_power": 24, "pa_gain": 1, "pwr_idx": 16},
+                {"rf_power": 25, "pa_gain": 1, "pwr_idx": 17},
+                {"rf_power": 26, "pa_gain": 1, "pwr_idx": 19},
+                {"rf_power": 27, "pa_gain": 1, "pwr_idx": 20}
+            ]
+        },
+        "radio_1": {
+            "enable": true,
+            "type": "SX1250",
+            "freq": 868500000,
+            "rssi_offset": -215.4,
+            "rssi_tcomp": {"coeff_a": 0, "coeff_b": 0, "coeff_c": 20.41, "coeff_d": 2162.56, "coeff_e": 0},
+            "tx_enable": false
+        },
+        "chan_multiSF_0": {
+            "enable": true,
+            "radio": 1,
+            "if": -400000
+        },
+        "chan_multiSF_1": {
+            "enable": true,
+            "radio": 1,
+            "if": -200000
+        },
+        "chan_multiSF_2": {
+            "enable": true,
+            "radio": 1,
+            "if": 0
+        },
+        "chan_multiSF_3": {
+            "enable": true,
+            "radio": 0,
+            "if": -400000
+        },
+        "chan_multiSF_4": {
+            "enable": true,
+            "radio": 0,
+            "if": -200000
+        },
+        "chan_multiSF_5": {
+            "enable": true,
+            "radio": 0,
+            "if": 0
+        },
+        "chan_multiSF_6": {
+            "enable": true,
+            "radio": 0,
+            "if": 200000
+        },
+        "chan_multiSF_7": {
+            "enable": true,
+            "radio": 0,
+            "if": 400000
+        },
+        "chan_Lora_std": {
+            "enable": true,
+            "radio": 1,
+            "if": -200000,
+            "bandwidth": 250000,
+            "spread_factor": 7
+        },
+        "chan_FSK": {
+            /* disabled */
+            "enable": false,
```

Comment:
Conflict with docs -- currently our published region plan for EU868 says that we listen for FSK on 868.8; this config doesn't enable it. Who is correct?